src/ci/utils.py

```diff
@@ -380,23 +473,28 @@ def runAStyleCheck(moduleName):
     """
     foldersToScan = _getFoldersToAStyle(moduleName)
     astyleCommand = "astyle --options=ci/input/astyle.config --dry-run " + foldersToScan
-    out = subprocess.run(astyleCommand, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
+    out = subprocess.run(astyleCommand, stdout=subprocess.PIPE,
+                         stderr=subprocess.PIPE, shell=True)
 
     if out.returncode == 0 and not out.stderr:
         stdoutString = str(out.stdout)
+
         if (stdoutString.find("Formatted") != -1):
             printFail('One or more files do not follow the Coding Style convention.')
-            printFail('Execute astyle --options=ci/input/astyle.config moduleName/*.h moduleName/*.cpp for further information.')
+            printFail('Execute astyle --options=ci/input/astyle.config moduleName/*.h moduleName/*.cpp for further'
```

Comment:
This seems to be a legacy issue but I believe it won't print the failing module correctly.